src/test/java/org/gridsuite/securityanalysis/server/SecurityAnalysisControllerTest.java

```diff
@@ -275,7 +271,7 @@ public void runAndSaveTest() throws Exception {
         assertEquals(RESULT_UUID.toString(), resultMessage.getHeaders().get("resultUuid"));
         assertEquals("me", resultMessage.getHeaders().get("receiver"));
 
-        mvcResult = mockMvc.perform(get("/" + VERSION + "/results/" + RESULT_UUID + "/n-result"))
+        mvcResult = mockMvc.perform(get("/" + VERSION + "/results/" + RESULT_UUID + "/n-result/paged"))
```

Comment:
I think you forgot to remove /paged in the unit test

Reply:
Fixed, thanks